### PR TITLE
:sparkles: Add moviedetails and cast page

### PR DESCRIPTION
**Pull request by daebert**
Pages MovieDetails und Cast

### CODE REVIEW

File: src/app/pages/MovieDetails/MovieDetails.module.css [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  display: grid;
+  column-gap: 1em;
+  width: 100%;
+  grid-auto-flow: column;
+  overflow: overlay;
+}
+
+.navBar {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr 1fr;
```
Comment (lajanzen) [inline src/app/pages/MovieDetails/MovieDetails.module.css]:
```suggestion
  grid-template-columns: repeat(4, 1fr);
```

Comment (Niklas-Schaefers) [inline src/app/pages/MovieDetails/MovieDetails.module.css]:
```suggestion
  grid-template-columns: repeat(4, 1fr);
```

**Review by lajanzen — CHANGES_REQUESTED** [review]
Lgtm, just one small change 🤓🚀

File: src/app/pages/Cast/Cast.module.css [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+.movieCardTitle {
+  font-weight: 600;
+}
+
+.movieCards {
+  display: grid;
+  column-gap: 1em;
+  width: 100%;
+  grid-auto-flow: column;
+  overflow: overlay;
```
Comment (phlgr) [inline src/app/pages/Cast/Cast.module.css]: This is deprecated and should be changed to scroll:
https://developer.mozilla.org/en-US/docs/Web/CSS/overflow#values

File: src/app/pages/MovieDetails/MovieDetails.module.css [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+.artistCardTitle {
+  font-weight: 600;
+}
+
+.artistCards {
+  display: grid;
+  column-gap: 1em;
+  width: 100%;
+  grid-auto-flow: column;
+  overflow: overlay;
```
Comment (phlgr) [inline src/app/pages/MovieDetails/MovieDetails.module.css]: https://developer.mozilla.org/en-US/docs/Web/CSS/overflow#values

File: src/app/pages/MovieDetails/MovieDetails.tsx [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+
+function MovieDetails(): JSX.Element {
+  return (
+    <div className={styles.container}>
+      <header className={styles.header}>
+        <BackButton></BackButton>
+      </header>
+      <main className={styles.main}>
+        <div className={styles.MovieTrailer}>
+          {/* <<MovieTrailer/>> */}
```
Comment (phlgr) [inline src/app/pages/MovieDetails/MovieDetails.tsx]: Could be removed

**Comment by l-goede** [comment]
very nice work!

**Review by phlgr — CHANGES_REQUESTED** [review]

File: src/app/pages/Cast/Cast.tsx [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+            <MovieCard imgSrc="./endOfWatch.png" />
+            <MovieCard imgSrc="./americanHustle.png" />
+            <MovieCard imgSrc="./crash.png" />
+          </div>
+        </div>
+      </main>
+      <footer className={styles.footer}>
+        <div className={styles.navBar}>
+          <NavBar />
+        </div>
```
Comment (Dev-dfm) [inline src/app/pages/Cast/Cast.tsx]: div not really needed. Could also be styled by the footer class

**Review by dhnazarenko — APPROVED** [review]
lgtm 🚀

**Review by Dev-dfm — COMMENTED** [review]
lgtm

**Review by Maurizio-F — COMMENTED** [review]
lgtm 💯

**Pull request merged**